Backpack-SwiftUI/SponsoredBanner/Classes/BPKSponsoredBanner.swift

```diff
@@ -150,6 +150,8 @@ public struct BPKSponsoredBanner: View {
                     .resizable()
                     .scaledToFit()
                     .frame(maxHeight: 22)
+                    .frame(maxWidth: 88)
```

Comment:
Can we use 2 constants maxLogoHeight and maxLogoWidth for these? This makes their purpose clearer